### ISSUE TITLE
Line artifacting when upscalling Guitar Hero: Warriors of Rock

### ISSUE THREAD

**Issue by kernaltrap8** [RPCS3/rpcs3#16271]
### Quick summary

When upscalling, GH:WoR has some minor, but noticeable artifacting on some parts of the viewport.
![image](https://github.com/user-attachments/assets/1f76674b-e5cb-40a3-9ac3-1fe4d421e537)
If I upscale to 4K, this artifacting goes away. It is not present when running at the native game resolution of 720p.
Look at the drumkit in this image. If you look at the right kick drum, there are some lines. These lines are the artifacting I am talking about. It is also visible too the right of the kit.

### Attach a log file

[RPCS3.log.gz](https://github.com/user-attachments/files/17594248/RPCS3.log.gz)

### Attach capture files for visual issues

[BLUS30487_20241031185439_capture.rrc.gz](https://github.com/user-attachments/files/17594308/BLUS30487_20241031185439_capture.rrc.gz)

### System configuration

CPU: AMD Ryzen 7 5800x
GPU: AMD Radeon RX570
OS: Gentoo Linux (6.11.4-gentoo-dist)

**Comment by Darkhost1999** [comment]
I haven't seen issues with upscaling accepted before on github. 150% values or non-integer values commonly have some very minor issues in games. While values of 200% and 300% are best. If you scale above your monitor/display resolution you only end up benefiting from SSAA. Yet if everything works at 720p resolution I have seen this as "not an emulation issue" for now. But I am sure someone official will re-evaluate.

**Comment by kernaltrap8** [comment]
> I haven't seen issues with upscaling accepted before on github. 150% values or non-integer values commonly have some very minor issues in games. While values of 200% and 300% are best. If you scale above your monitor/display resolution you only end up benefiting from SSAA. Yet if everything works at 720p resolution I have seen this as "not an emulation issue" for now. But I am sure someone official will re-evaluate.

I see it as a emulation issue if the artifacting only happens at certain resolutions.

**Comment by RainbowCookie32** [comment]
Unless it's a regression, issues from upscaling are not valid. The games were never meant to run beyond what they ran on the PS3 and may not play along. Upscaling is an enhancement that may or may not be compatible with the games you want to play

**Comment by kernaltrap8** [comment]
> Unless it's a regression, issues from upscaling are not valid. The games were never meant to run beyond what they ran on the PS3 and may not play along. Upscaling is an enhancement that may or may not be compatible with the games you want to play

I don't think you understand. At 720p, the artifacting isn't present. At any resolution before 4K that is above 720, the artifacting appears. It is not present 4K or above.

**Comment by alexconsole04** [comment]
The artifacts don't appear on native resolution because it's native resolution. As already said, upscaling the resolution will cause issues on some games since they were never supposed to run on higher resolutions, which as a result, will cause visual artifacts. Please slowly reread your statement and you'll see why it makes no sense for this GitHub issue to be a thing.

**Comment by Virtua64** [comment]
> The artifacts don't appear on native resolution because it's native resolution. 

But they dont appear at 4k either.  Anyway, try changing the Resolution Scale Threshold to 128x128 or 512 X 512 or even 1x1 to see if it resolves the issue.

**Comment by kd-11** [comment]
Closing as the issue is caused by running an upscaled resolution.
Upscaling doesn't work with all games and even with the ones that allow it, there are limitations.
